src/core/range.c

```diff
@@ -259,6 +259,7 @@ QuicRangeAddRange(
     *RangeUpdated = FALSE;
 
 #if QUIC_RANGE_USE_BINARY_SEARCH
+    // FIXME: runtime error: unsigned integer overflow: 0 - 1 cannot be represented in type 'unsigned int'
```

Comment:
This code relied on underflow. I think we can just annotate/ignore this function.